packages/document/docs/en/plugins/list/plugin-babel.mdx

```diff
@@ -56,7 +56,15 @@ const defaultOptions = {
   babelrc: false,
   compact: false,
   configFile: false,
-  plugins: [],
+  plugins: [
+    ['@babel/plugin-proposal-decorators', config.source.decorators],
+    ...(isLegacyDecorators
+      ? [
+          require.resolve('@babel/plugin-transform-class-properties'),
```

Comment:
The default value in the document can be made easier to read.

`require.resolve` can be omitted here.

Reply:
done